packages/core/src/navigation/navigation.stories.mdx

```diff
@@ -0,0 +1,86 @@
+import { Meta, Props, Story, Preview, API } from '@storybook/addon-docs/blocks';
+import { html, LitElement } from 'lit-element';
+import '@cds/core/navigation/register.js';
+
+<Meta title="Components/Navigation" />
+
+# Navigation
+
+A sound navigation layout offers a high degree of discoverability and feedback, letting users know where they are at all
+times and ensuring they can easily get to where they want to go.
+
+Navigation enables discoverability and feedback forms users. It lets them know where they are at all times and ensures
+they can easily get where they want to go.
+
+Core navigation includes features that enables developers to create multiple navigation layouts from one component that
```

Comment:
`Core navigation provides a foundation that will enable developers...`

Reply:
updated